### DISCUSSION TITLE
Release Rulesets to NuGet

### DISCUSSION THREAD

**Discussion by LeNitrous** [ppy/osu#16031] [Ideas] [answered]
I'm currently developing [sbtw](https://github.com/LeNitrous/sbtw), a tool which uses osu! lazer (from NuGet) allowing storyboarders to create scripts in C#, Visual Basic, Python, and Javascript and export it as a legacy osu! storyboard file (.osb). Currently there is no way to provide the default rulesets other than to directly take them from an existing lazer installation to the rulesets folder.

Another possible use case is to allow ruleset creators to extend the default rulesets by referencing them in their projects.

**Comment by peppy** [comment, marked as answer]
https://www.nuget.org/packages?q=ppy.osu.game.rulesets
